pkg/kubelet/kubelet_resources_test.go

```diff
@@ -38,6 +39,16 @@ func TestPodResourceLimitsDefaulting(t *testing.T) {
 	}, nil)
 	tk.fakeCadvisor.On("ImagesFsInfo").Return(cadvisorapiv2.FsInfo{}, nil)
 	tk.fakeCadvisor.On("RootFsInfo").Return(cadvisorapiv2.FsInfo{}, nil)
+
+	// Resetting default cpu and memory reservation to 0, so that
+	// node capacity is same as node allocatable.
+	tk.kubelet.reservation = kubetypes.Reservation{
+		Kubernetes: api.ResourceList{
+			api.ResourceCPU:    resource.MustParse("0"),
```

Comment:
having the code structured like this obfuscates that its pulling allocatable that is not equal to capacity, so prefer the test to set the reservation (ideally for Kubernetes and System).